gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -17,6 +17,8 @@
 import argparse  # pylint: disable=unused-import
 import re
 
+from apache_beam.io.filesystem import BeamIOError
```

Comment:
nit: please do `from apache_beam.io import filesystem` and `from apache_beam.io import filesystems` and then refer to `filesystem.BeamIOError` below (it's a google style thing to not import classes as much as possible).